manager/aica_django/converters/Knowledge.py

```diff
@@ -845,31 +875,52 @@ def nmap_scan_to_knowledge(
 
     knowledge_nodes = []
     knowledge_relations = []
+    if "runtime" not in scan_results:
+        print("gotcha!", scan_results)
```

Comment:
This message isn't helpful, can it be made more informative? It should also use the logger instead of print.

Reply:
this will be removed, was for debugging...

Reply:
Okay, please go ahead and remove. Thank you!

Reply:
Thanks, please go ahead and remove.